roles/calico/templates/calico.service.j2

```diff
@@ -10,8 +10,12 @@ ExecStart=/usr/bin/docker run --net=host --privileged \
  --name=calico-node \
  -e WAIT_FOR_DATASTORE=true \
  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
- -e CALICO_IPV4POOL_IPIP=always \
- -e FELIX_IPV6SUPPORT=false \
+ -e CALICO_IPV4POOL_IPIP={{ calico_ipv4pool_ipip }} \
+ -e CALICO_IPV4POOL_CIDR={{ calico_ipv4pool_cidr }} \
```

Comment:
Maybe
```
{% if calico_ipv4pool_cidr is defined %}
-e CALICO_IPV4POOL_CIDR={{ calico_ipv4pool_cidr }} \
{% endif %}
```

and comment the `calico_ipv4pool_cidr: "192.168.0.0/16"` in the `main.yml`?